docs/users-guide/account-settings.md

```diff
@@ -0,0 +1,24 @@
+## Account settings
+
+You can view account settings by clicking on the **gears** icon in the upper right of the main navigation bar, then selecting **Account settings**. Here you'll find three tabs:
+
+- [Profile](#account-profile)
+- [Password](#account-password)
+- [Login history](#account-login-history)
+
+### Account profile
+
+You can set your first and last names, change your email address, and set your language. See our list of [supported languages](../faq/general/what-languages-can-be-used-with-metabase.md#currently-available-languages).
+
+### Account password
+
+You can change your password here. If you're having trouble logging in, see our [How do I reset my password](../faq/using-metabase/how-do-i-reset-my-password.md).
+
+### Account login history
+
+The login history lists each login, along with some location information (if available), and some client information (like Browser (Firefox/Windows)).
+If you see any suspicious login attempts, change your password and notify your administrator.
+
+#### A note about new login emails
+
+Whenever you login from a new device, Metabase will send you an email just to let you know someone (hopefully you) has logged in from an unrecognized device. If you see this email, but don't remember logging in, or don't recognize the device, change your password and let your administrator know. 
```

Comment:
Verb rather than noun in this case.

```suggestion
Whenever you log in from a new device, Metabase will send you an email just to let you know someone (hopefully you) has logged in from an unrecognized device. If you see this email, but don't remember logging in, or don't recognize the device, change your password and let your administrator know. 
```